jdi-light-applitools-tests/src/main/java/jdisite/entities/ContactInfo.java

```diff
@@ -0,0 +1,14 @@
+package jdisite.entities;
+
+/**
+ * Created by Roman Iovlev on 10.11.2018
+ * Email: roman.iovlev.jdi@gmail.com; Skype: roman.iovlev
+ */
+
+import com.epam.jdi.tools.DataClass;
+
+public class ContactInfo extends DataClass<ContactInfo> {
+    public String name, lastName, position, gender, religion, weather, description;
+    public int passportNumber, passportSeria;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5408219)